posthog/warehouse/api/test/test_external_data_source.py

```diff
@@ -371,7 +371,8 @@ def test_list_external_data_source(self):
         self._create_external_data_source()
         self._create_external_data_source()
 
-        response = self.client.get(f"/api/projects/{self.team.pk}/external_data_sources/")
+        with self.assertNumQueries(17):
```

Comment:
Do you know what this number was before?

Reply:
Was 20 something. Each extra source was adding 4 or so queries